common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala

```diff
@@ -40,6 +37,10 @@ import java.time.Instant
 import scala.collection.immutable.SortedMap
 
 object ObsQueries {
+  implicit class ListOps[A](val list: List[A]) {
```

Comment:
extension method? 
We have this someplace common don't we? I assume it isn't visible from here?

Reply:
We do but it maybe hard to have it in common so I copied it